CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - EXPERIMENTAL: (#382) Working copy snapshots are created before potentially-destructive operations in order to improve the capabilities of `git undo`.
   - Working copy snapshots are taken by default. Disable by setting `branchless.undo.createSnapshots` to `false`.
 - EXPERIMENTAL: (#391) created the `git record` command, which opens an interactive change selector for committing.
+- Numerous updates to `git branchless move`:
+  - (#400) Added `--insert` option to to insert the moved subtree or commits into the commit tree between the destination commit and it's children, if any.
+  - (#450) Added `--exact` option to move a specific set of commits elsewhere in the commit tree. They will be removed from their current location and any unmoved children (if any) will be moved to their closest unmoved ancestor. The moved commits will maintain their overall ancestry structure.
+  - (#447) `--source`, `--base`, `--range` and `--exact` options can all be provided multiple times and in any combination to move multiple subtress, ranges and/or commits to the destination.
```

Comment:
- I think we no longer have the `--range` flag.
- I think these flags can be provided multiple times, but not in arbitrary combinations.

Reply:
Yup, I totally missed that. Sorry, and thanks for catching it!

Reply:
Not for this PR, of course, but I do wonder if the current implementation would actually support "arbitrary combinations" of these flags if we just removed the clap `conflicts_with` attributes. (Or if it could do so w/ minimal changes.) I don't know that it would be *useful* to do so, though...